bot/utils/helpers.py

```diff
@@ -62,7 +62,10 @@ def predicate(message: discord.Message) -> bool:
             return thread
 
     logger.info("Message not found in either cache, fetching all threads...")
-    for thread in await channel.active_threads():
+    for thread in await channel.guild.active_threads():
+        if thread.parent != channel:
+            continue
+
```

Comment:
I recently found this out, could this implementation detail optimize the method?

> The created thread and the message it was started from will share the same id.

[Discord Developer Portal - Documentation - Threads](https://discord.com/developers/docs/topics/threads#public-private-threads)

Reply:
Hmmm, yea we could. I'd like to investigate that outside of this PR though, since #4 is based on top of this PR and I CBA to mess with git right now lol